src/app/server/config/day1.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "appType": "basic",
+  "title": "Day 1 - Basic Model",
+  "basicProp": "day1 basic value"
```

Comment:
Just confirming here that all the `xxxxxProp` keys are placeholders for some future arbitrary data?

Reply:
Yes. I'd expect something `appType` and `title` to to remain though. As we discussed, there isn't actually that much to distinguish the app type configs from each other though - I made some notes on that here: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2913